Gandalan.IDAS.WebApi.Client/BusinessRoutinen/WebRoutinenBase.cs

```diff
@@ -83,6 +91,11 @@ protected virtual void OnErrorOccured(ApiErrorArgs e)
         /// <returns>Status des Logins</returns>
         public bool Login()
         {
+            if (IsJwt)
```

Comment:
Validity check neccessary for expired tokens?

Reply:
Added JWT basic token validation (expired?) and call to `api/LoginJwt/Refresh` if `refreshToken` is available